oauth2client/contrib/gce.py

```diff
@@ -85,32 +138,128 @@ class AppAssertionCredentials(AssertionCredentials):
     information to generate and refresh its own access tokens.
     """
 
-    @util.positional(2)
-    def __init__(self, scope='', **kwargs):
+    def __init__(self,
+                 scope=None,
+                 service_account_email='default',
```

Comment:
The metadata server provides a "default" service account so that users don't have to know the email of the default service account in order to fetch access tokens. To use the default service account, you supply the string 'default' as the `service_account_email` to the metadata server. For backwards compatibility, (where specifying a specific service account is not possible, and this class always uses the default service account), I need to be able to leave this out. 

I could do

```
def __init__(..., service_account_email=None):
    if service_account_email is None:
        service_account_email='default'
```

But I don't see any advantage to that. Do you feel the doc string needs to be clearer? Do you have a suggestion for prose?